README.rst

```diff
@@ -26,7 +26,7 @@ The installation instructions for this extension is available
 from the `Celery documentation`_
 
 .. _`Celery documentation`:
-    https://docs.celeryproject.org/en/latest/django/first-steps-with-django.html#django-celery-results-using-the-django-orm-cache-as-a-result-backend
+    https://docs.celeryq.dev/en/latest/django/first-steps-with-django.html#django-celery-results-using-the-django-orm-cache-as-a-result-backendresults-using-the-django-orm-cache-as-a-result-backend
```

Comment:
can you please fix the duplication here?

Reply:
Oops, my apologies. Fix committed.